Extensions/AzureExp/Src/UIContribution/src/ExperimentOverview.tsx

```diff
@@ -0,0 +1,207 @@
+import './ExperimentOverview.scss';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as Release from 'azure-devops-extension-api/Release';
+import * as SDK from 'azure-devops-extension-sdk';
+import ProgressionComponent from './Component/ProgressionComponent';
+import ExpRestClient from './ExpRestClient';
+import ExpUtility from './ExpUtility';
+import { LoadingSpinner } from './Component/LoadingSpinner';
+import { MessageCard, MessageCardSeverity } from 'azure-devops-ui/MessageCard';
+import { Card, ICardTitleProps } from 'azure-devops-ui/Card';
+import { TitleSize } from 'azure-devops-ui/Header';
+import { Link } from 'azure-devops-ui/Link';
+import { Icon } from 'azure-devops-ui/Icon';
+import { Table, renderSimpleCell, ITableColumn, SimpleTableCell } from 'azure-devops-ui/Table';
+import { ObservableValue } from 'azure-devops-ui/Core/Observable';
+import { ArrayItemProvider } from 'azure-devops-ui/Utilities/Provider';
+
+export interface IExperimentOverviewState {
+    progression: any;
+    feature: any;
+    scorecards: any;
+    loaded: boolean;
+    error?: string;
+}
+
+export class ExperimentOverview extends React.Component<{}, IExperimentOverviewState> {
+    constructor(props) {
+        super(props);
+        this.state = {
+            loaded: false,
+            progression: null,
+            scorecards: null,
+            feature: null
+        };
+
+        this._expClient = new ExpRestClient();
+    }
+
+    public componentDidMount() {
+        SDK.register('registeredEnvironmentObject', {
+            isInvisible: (context): boolean => {
+                return !ExpUtility.isOverviewTabVisible(context);
+            }
+        });
+        
+        SDK.init();
+        SDK.ready().then(() => {    
+            let context = SDK.getConfiguration();
+            console.log(`Extension context`, context);
+            if (!!context.releaseEnvironment) {
+                this._expClient.getRelease(context.releaseEnvironment.releaseId).then((release: Release.Release) => {
+                    let [featureId, progressionId] = ExpUtility.getFeatureAndProgressionIdFromRelease(release);
+                    if (!featureId || !progressionId) {
```

Comment:
Should we show the UI if there is only featureId? Provisioning progression step not be there or may fail due to some reason.

Reply:
Modified to show Feature info and feature flag details if only featureId is available.

Reply:
Modified to show feature details and feature flag info if only featureId is available.